extensions/podman/packages/extension/src/checks/windows/hyperv-running-check.spec.ts

```diff
@@ -0,0 +1,67 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import type { TelemetryLogger } from '@podman-desktop/api';
+import { beforeEach, expect, test, vi } from 'vitest';
+
+import type { PowerShellClient } from '/@/utils/powershell';
+import { getPowerShellClient } from '/@/utils/powershell';
+
+import { HyperVRunningCheck } from './hyperv-running-check';
+
+vi.mock(import('@podman-desktop/api'));
+vi.mock(import('../../utils/powershell'), () => ({
+  getPowerShellClient: vi.fn(),
+}));
+
+const mockTelemetryLogger = {} as TelemetryLogger;
```

Comment:
remark: You are using uppercase at L33, but here lowercase, I know there is debate, but as this is a new file, might be nice to have a single logic (I do prefer uppercase with `_MOCK` suffix)

Reply:
it is a copy and paste, we will see to uniformize later